Mathlib/LinearAlgebra/TensorProduct/Quotient.lean

```diff
@@ -172,6 +173,17 @@ noncomputable def quotTensorEquivQuotSMul (I : Ideal R) :
       simp only [Set.mem_image, Set.mem_setOf_eq]
       exact ⟨r ⊗ₜ m, ⟨r, hr, m, rfl⟩, rfl⟩)
 
+/-- As a module of the quotient ring, left tensoring a module with a quotient of the ring
+is the same as quotienting that module by the corresponding submodule. -/
+noncomputable def quotTensorEquivQuotSMul' (I : Ideal R) :
+    ((R ⧸ I) ⊗[R] M) ≃ₗ[R ⧸ I] (M ⧸ I • (⊤ : Submodule R M)) :=
+AddEquiv.toLinearEquiv (quotTensorEquivQuotSMul M I).toAddEquiv <| by
+    intro c x
+    refine Quotient.inductionOn c (fun c ↦ ?_)
+    show (quotTensorEquivQuotSMul M I).toLinearMap (c • x) =
+      c • (quotTensorEquivQuotSMul M I).toLinearMap x
+    rw [LinearMap.CompatibleSMul.map_smul]
```

Comment:
```suggestion
(quotTensorEquivQuotSMul M I).extendScalarsOfSurjective Ideal.Quotient.mk_surjective
```
(not tested)

Reply:
If this works I'm inclined to not add this def.
If you still think this is necessary, this probably should be moved to somewhere else to avoid the `large-import`.

Reply:
Great, thanks! (LinearEquiv.extendScalarsOfSurjective was not yet in our mathlib version or I didn't see it).